### PR TITLE
add postgres

### PR DESCRIPTION
**Pull request by ynachiket**

### CODE REVIEW

**Comment by lekhajee** [comment]
Should this go under the directory `deps/`

**Comment by robert-chiniquy** [comment]
> Should this go under the directory `deps/`

Yes

**Comment by ynachiket** [comment]
We are not using it off the shelf. We need to do a bunch of scaffolding .. look at the postgres/sql dir.

Also the dockerfile needs to be in the dir for context sharing.

**Comment by ynachiket** [comment]
@robert-chiniquy @lekhajee fixed in 68bd95b

File: deps/postgres/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# And add ``listen_addresses`` to ``/etc/postgresql/9.3/main/postgresql.conf``
+RUN echo "listen_addresses='*'" >> /etc/postgresql/9.3/main/postgresql.conf
+
+# Expose the PostgreSQL port
+EXPOSE 5432
+
+# Add VOLUMEs to allow backup of config, logs and databases
+VOLUME  ["/etc/postgresql", "/var/log/postgresql", "/var/lib/postgresql"]
+
+# Set the default command to run when starting the container
```
Comment (robert-chiniquy) [inline deps/postgres/Dockerfile:43]: do we need a running postgres instance for this project?

Comment (ynachiket) [inline deps/postgres/Dockerfile:43]: the noit conf that I have needs it and fails to start without it.

Comment (robert-chiniquy) [inline deps/postgres/Dockerfile:43]: cool fine for now

**Comment by robert-chiniquy** [comment]
+1

**Pull request merged**